mindmeld/components/entity_recognizer.py

```diff
@@ -141,6 +140,15 @@ def dump(self, model_path, incremental_model_path=None):
         )
         super().dump(model_path, incremental_model_path)
 
+    def unload(self):
+        logger.info(
+            "Unloading entity recognizer: domain=%r, intent=%r", self.domain, self.intent
+        )
+        self.entity_types = None
```

Comment:
Do we also want to unload the reference to the resource loader?

Reply:
There are probably some properties retaining memory that I've missed.  Please let me know if you notice any.

Reply:
As we discussed, since there are many refs to the resource loader, it doesnt make sense to unload it